src/bundle/SystemInfo/Collector/IbexaSystemInfoCollector.php

```diff
@@ -51,12 +52,12 @@ class IbexaSystemInfoCollector implements SystemInfoCollector
      *
      * @see: https://support.ibexa.co/Public/Service-Life
      */
-    const EOM = [
+    public const EOM = [
         '2.5' => '2022-03-29T23:59:59+00:00',
         '3.0' => '2020-07-10T23:59:59+00:00',
         '3.1' => '2020-11-30T23:59:59+00:00',
         '3.2' => '2021-02-28T23:59:59+00:00',
-        '3.3' => '2021-04-30T23:59:59+00:00', // Estimate at time of writing
+        '3.3' => '2022-12-30T23:59:59+00:00', // Estimate at time of writing
```

Comment:
Afaik our LTS support policy is 3+2: https://support.ibexa.co/Public/Service-Life

```suggestion
        '3.3' => '2023-12-30T23:59:59+00:00', // Estimate at time of writing
```

Reply:
> Afaik our LTS support policy is 3+2 (so total 5 years): https://support.ibexa.co/Public/Service-Life

Ah, I misunderstood you initially. Fixed via 047e786.